src/cookbooks/clients_last_seen_bits.md

```diff
@@ -0,0 +1,136 @@
+# Working with Bit Patterns in Clients Last Seen
```

Comment:
You may already have been thinking about this, but I think we need some kind of introductory paragraph(s) to motivate this content. Describing some concrete questions that might be answered by this material would be good.